ViewModel/HyvaGrid/Navigation.php

```diff
@@ -26,6 +26,8 @@ class Navigation implements NavigationInterface
 
     private SearchCriteriaBuilder $searchCriteriaBuilder;
 
+    protected SearchCriteriaInterface $memorizedSearchCriteria;
```

Comment:
Please don't use protected properties.
The name should be `memoize` not `memorize` (without the r), reference https://en.wikipedia.org/wiki/Memoization

Reply:
Thanks about that. Looks like I used that wrong for a looong time :)